sdk/communication/communication-call-automation/test/utils/recordedClient.ts

```diff
@@ -43,8 +41,9 @@ import type {
 import { ServiceBusClient } from "@azure/service-bus";
 import type { PhoneNumbersClientOptions } from "@azure/communication-phone-numbers";
 import { PhoneNumbersClient } from "@azure/communication-phone-numbers";
+import { assert, vi } from "vitest";
 
-if (isNode) {
+if (isNodeLike) {
   dotenv.config();
```

Comment:
doesn't the recorder handle this now?

Reply:
It probably does, this is just a straight forward translation.